src/pages/Quiz/Quiz.tsx

```diff
@@ -53,13 +52,22 @@ export const Quiz = () => {
 
   const handleAnswerClick = (answerIndex: number) => {
     const result = question?.correctAnswerIndex === answerIndex;
-    // console.log(result);
     setQuizState(result ? QuizState.Correct : QuizState.Incorrect);
     if (result) {
       setScore((prev) => prev + 1);
     }
   };
 
+  useEffect(() => {
+    setQuizState(QuizState.Question);
+  }, [questionNumber]);
+
+  const isLoading = quizQuestionIsLoading || quizListIsLoading;
+
+  if (isLoading) {
+    return <div className={classes.loading}>Loading...</div>;
```

Comment:
Tady to přepiš asi na "Načítání...". Já si včera neuvědomil, že je ta appka v češtině :D